### PR TITLE
gputest

### PR DESCRIPTION
**Pull request by SeijiEmery**
Adds a basic opengl test based off of render-utils-test, but using the gpu:: library and with no raw gl calls.

### CODE REVIEW

File: tests/gpu-test/src/gputest_simple_vert.h [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+in vec4 inTexCoord1;
+
+
+
+
+
+
+
+
+
```
Comment (AndrewMeadows) [inline tests/gpu-test/src/gputest_simple_vert.h]: So

much

whitespace!

**Comment by hifi-gustavo** [comment]
Build console output for commit: dbf94d673be4b128ce510d45fcada3c27b1567e5 available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

File: tests/gpu-test/src/gputest_simple_vert.h (hunk 1 above)
Comment (SeijiEmery) [inline tests/gpu-test/src/gputest_simple_vert.h]: this is autogenerated, so don't blame me xD

File: tests/gpu-test/src/gputest_simple_vert.h [hunk 2]
```diff
@@ -0,0 +1,113 @@
+// File generated by Scribe Wed Aug  5 16:50:23 2015
```
Comment (samcake) [inline tests/gpu-test/src/gputest_simple_vert.h]: You don;t need to add this file to the repository, it gets created from the sl* version
you have to remove that file

File: tests/gpu-test/src/gputest_simple_frag.h [hunk 3]
```diff
@@ -0,0 +1,105 @@
+// File generated by Scribe Wed Aug  5 16:50:24 2015
```
Comment (samcake) [inline tests/gpu-test/src/gputest_simple_frag.h]: You don;t need to add this file to the repository, it gets created from the sl* version
you have to remove that file

**Comment by PhilipRosedale** [comment]
this is failing?  Can this get fixed?

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: dbf94d673be4b128ce510d45fcada3c27b1567e5 available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by ZappoMan** [comment]
This is failing on windows... looks like an easy fix.... @SeijiEmery - can you please push a fix to this so we can get this PR merged.

```
"D:\jenkins\workspace\pull-request-win\build\tests\gpu-test\gpu-test.vcxproj" (default target) (87) ->
       (ClCompile target) -> 
         D:\jenkins\workspace\pull-request-win\libraries\animation\src\AnimationCache.h(15): fatal error C1083: Cannot open include file: 'QScriptEngine': No such file or directory (..\..\..\tests\gpu-test\src\main.cpp) [D:\jenkins\workspace\pull-request-win\build\tests\gpu-test\gpu-test.vcxproj]
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6b5fb60fcf0bbed95e91c842959c68a6efeab1c5 available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e601484b17ff8219ab94ce79410a8cb6e7864c3a available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

File: tests/gpu-test/src/gputest_simple_frag.h (hunk 3 above)
Comment (samcake) [inline tests/gpu-test/src/gputest_simple_frag.h]: Seiji, you need to remove this file from git, it s autogenerated

File: tests/gpu-test/src/gputest_simple_vert.h (hunk 2 above)
Comment (samcake) [inline tests/gpu-test/src/gputest_simple_vert.h]: Seiji, you need to remove this file from git, it s autogenerated

**Comment by hifi-gustavo** [comment]
Build console output for commit: 691cb48a08af1f36aeb0707aac17e2c0d27d5fac available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: bd18951be81cc33b608f0c4255639f42b066a19d available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by SeijiEmery** [comment]
test this please

**Comment by SeijiEmery** [comment]
...no idea why this is failing. Builds fine in xcode and make on my machine.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 316ddd68ef6b7b2d355dae0982aa0a44d3688326 available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3d59f8d7e8279c673272df4bca1b7a74339bfe7b available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 62dc83fa32ac1fb9b8bfdcfaf6870348b4adf9dc available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 248d0729a62834f9560ea280df39f8847508b8b9 available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 140edd0589e331f46bbdae88441f92bfb28e5938 available at: https://gist.github.com/fc9b3f7d7d57de5169a0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5529.dmg

**Pull request merged**